### ISSUE TITLE
After recently update, no icon shown for node_module folder

### ISSUE THREAD

**Issue by MehdiDehghani** [robertohuertasm/vscode-icons#207]
Hello,

I googled this, but I found nothing about that, so I reported.
The extension works perfectly for every files & folders, but `node_modules`. is this new behavior or is something wrong?

I'm using latest version of `VSCode` & your extension. also `Windows 10 64bit`.

Thanks for your great extension btw !

**Comment by robertohuertasm** [comment]
Have you tried executing 'icons update'?

**Comment by robertohuertasm** [comment]
Do see the normal folder icon or nothing? I've tested in my Windows 10 and it seems to be working. Plus, the icons.zip currently deployed contains the proper folder icons.

**Comment by MehdiDehghani** [comment]
Yes, I tired `Icon update` command (*in admin mode*), and yes, as I said icons shown perfectly for other files & folders.

I should mentioned the folder icon doesn't show for `.vscode' folder too.
Seems icon allocated the space, but there is no icon for this 2 guys.

**Comment by robertohuertasm** [comment]
Have you tried recently? Icons update will reinject the custom code and download again the icons.zip file from this repo. As you can see the icons.zip contains all the folder icons so... My only guess is that you're using a proxy or that you downloaded a bad icons.zip. 

When updating do you see any message in your dev console?

**Comment by MehdiDehghani** [comment]
I tired update command just before posting my answer to your question.
Why other folders have `folder icon` but this 2 folders?

**Maybe this help**

The icon for `node_module` was correct, but when I hit update (*in extension section of vscode*), and of course after restating vscode, I found out the icon of `package.json` changed to `npm` icon (*the red one*). after this update the folder icon of `node_module` disappeared.

**Comment by MehdiDehghani** [comment]
BTW here is the screenshot of dev console after I run Update command.
![sss](https://cloud.githubusercontent.com/assets/9152330/18033953/73202aac-6d45-11e6-9be9-f0a0a94e8c85.png)

**Comment by robertohuertasm** [comment]
@MehdiDehghani everything seems fine. Try this:

1. `icons disable`
2. uninstall the extension
3. install it again

I know that you have done it already but just for the sake of discarding obvious things as know I've checked the `icons.zip` file and seems to be ok. I've tested it in a clean installation and it's working so it should work for you too. It has no sense that only two icons are not present if they're all in the package.
